src/shogun-core-main/src/test/java/de/terrestris/shoguncore/web/ImageFileControllerTest.java

```diff
@@ -50,12 +48,9 @@ public void getThumbnailReturnsErrorMapIfIdIsNull() throws Exception {
 
         MvcResult result = mockMvc.perform(MockMvcRequestBuilders.get(GET_THUMBNAIL_ENDPOINT)
             .param("id", ""))
-            .andExpect(status().isOk()).andReturn();
+            .andExpect(status().is4xxClientError()).andReturn();
 
-        assertEquals(result.getResponse().getStatus(), 200);
-        ObjectMapper objectMapper = new ObjectMapper();
-        assertEquals("Returned body matched mocked one.",
-            objectMapper.writeValueAsString(ResultSet.error("Could not get the image thumbnail: Could not find the image with id null")), result.getResponse().getContentAsString());
+        assertEquals(result.getResponse().getStatus(), 400);
```

Comment:
This looks a bit as if the API has changed after the upgrades. Is this correct? Would this require a major release?

Reply:
See below, it's the same issue